bitcoin/src/psbt/mod.rs

```diff
@@ -1187,6 +1233,7 @@ mod tests {
                 // This weird thing is necessary since rustc 0.29 prints out I/O error in a different format than later versions
                 .map_err(|err| match err {
                     PsbtParseError::PsbtEncoding(err) => err,
+                    PsbtParseError::Version(_) => panic!("PSBT Base64 decoding failed"),
```

Comment:
This should say "invalid version". (The whole thing looks weird but I'm not in the mood to figure out a better way.)